authorPatchCollector.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+
+total_parameters = len(sys.argv)
+
+if total_parameters != 3:
+    print(f'Usage: {str(sys.argv[0])} author_name output_root_directory')
+    print()
+    print("This is a simple helper script that allows collecting all patches from")
+    print("one author in a git repository. It will detect consequent ranges of")
+    print("patches and ask for each range for a subdirectory so you can distinguish")
+    print("the patch series'.")
+    sys.exit()
+
+author = str(sys.argv[1])
+output = str(sys.argv[2])
```

Comment:
What about using `argparse`?

Reply:
That will be more appropriate, I guess. Thanks :)